### DISCUSSION TITLE
Which AWS program should I choose?

### DISCUSSION THREAD

**Discussion by Ohado** [l3atbc-datadog/pushkin#178] [Q&A]
Hi.
I am new to Pushkin, and I try to follow the tutorial. I see AWS has a lot of options, as I can see here: https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc
Which option is the one that's recommended for Pushkin?

**Comment by jkhartshorne** [comment]
@Ohado welcome to Pushkin! I apologize for leaving you hanging for so long. Unfortunately, you wrote right around the time the funding ran out for the full-time assistant I had for doing user support! If you follow [the Pushkin tutorial](https://languagelearninglab.gitbook.io/pushkin/), it'll tell you exactly what to use.

I should warn you that while our CLI will do a lot of the setup for you, some parts are still a bit finicky. Please ask any questions here, and I'll try to get back to you as soon as possible.

**Reply by Ohado** [reply]
Thank you. I am following the tutorial, but got stuck when it tells me to "specify a database to use" on the "config.yaml". I'm afraid I don't understand what's needed to be done there, and leaving it as it is seems to cause a problem when doing "pushkin prep", causing a database start problem.

**Comment by jkhartshorne** [comment]
Sorry for the late reply. The default config.yaml should already
have something specified. Are you working on the external deploy, or trying
to deploy it locally on your own computer?
